domains/apurboislam.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "apurboislam",
+           "email": "",
+           "discord": "740207602855575565",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.Tb7jeRGTM3baPJ66NxXky5UUpue3qxinQq5Fq_qYbhEojPyOjmyTJopWxZGfTIie-42dA5RE5sEGUWU3YTEtXq8jvyQA2Alhv0Mz4fHfbfb7CKClfuoBT-_gJBxhrIRtOzfAXMkBofshbSTk8n8w0fcNaFRNn61YFjpULCdAJ_mf7ZGU6wmUxPvOdejIMskSXYPif0rQSk3DO2WHPxQO17fUPDaU3Q0u52CTTStdyyH3Efrj_DV7_wStGocc0AQQ2lUXxn_bPEJ1f8PzgVUk8AYbc0jQjQeokjtcRda4IIWn01CJ2s23xKJ1VW6ytPTYh91K72RINOHcFrY4miFyfA.EtMgYVQwI8Fkim2yde90bg.opKL6p-2Wkm2NpUIpUZD64bRAaGxY_PujYozpSQd24hSw_tEYcSsWR2LB43u-Whzv3ILzUyQIYAXT-q_W_ZP0dESFauT2szYJ759LjVconDiQPYpb7Y6M7K9ybJRkXDuDTgyx9QPBjG8uqdSRxQ3cQ.NrG09Bz5PZwk5xtQ77MELA"
+        },
+    
+        "record": {
+            "CNAME": "apurboislam.github.io"
```

Comment:
You should complete your website first, in the [repo](https://github.com/apurboislam/is-a.dev/) i only saw a "Hello world"

Reply:
Okay, i will!

Reply:
If i redirect to my another website does it break rules?